src/test/travis_run.sh

```diff
@@ -103,6 +103,14 @@ if test "$CPPCHECK" = "t"; then
     sh -x src/test/cppcheck.sh
 fi
 
+# Launch the MUNGE daemon
+echo "Starting MUNGE"
+case $BASE_IMAGE in
+    bionic*) sudo /etc/init.d/munge start ;;
+    centos*) sudo /sbin/runuser -u munge /usr/sbin/munged ;;
```

Comment:
This looks like it will work fine, but I'm curious about the differences.  I would have thought `systemctl start munge` would work on both, or `sudo -u munge /usr/sbin/munged` for that matter.  Any particular reason for the split?

Reply:
For whatever reason, passing the `-u` flag to sudo on bionic-base causes it to prompt for a password when building the image (at least on my local machine).  And for whatever other reason, passing the `-u` flag to sudo on centos7-base causes `/var/tmp/sclfh58sd: line 8: -u: command not found`.  :man_shrugging: 

I was able to remove the `case` command and just always use `sudo /sbin/runuser -u munge /usr/sbin/munged` by first creating the `/var/run/munge` directory (with owner `munge`) in the bionic-base image.